gomod-sync/.gitignore

```diff
@@ -0,0 +1,15 @@
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+# vendor/
```

Comment:
Do we really need a separate gitignore file here? Couldn't we update the existing root level file instead?

Reply:
I thought about it and I don't have a strong opinion. In the end I decided to put it here because I didn't want to pollute the global gitignore, and helps this folder to me more self-contained - we can move the folder around without the need to update the gitignore.

But I don't have a strong opinion either way.

Reply:
I thought about it and I don't have a strong opinion. In the end I decided to put it here because I didn't want to pollute the global gitignore, and helps this folder to me more self-contained - we can move the folder around without the need to update the gitignore.

But I don't have a strong opinion either way.

Reply:
If you don't mind it would be great if you could remove this file and update the root gitignore instead. Imo, this is not polluting it but instead this file here mostly contains the standard entries that should be in the root level file anyway.

Also please include "gomod-sync" in the list of files to ignore so the binary that gets build does not show up in the files to commit.

Reply:
Done.